README.md

```diff
@@ -5,6 +5,11 @@ A simple package manager for the Motoko programming language.
 ## Getting started
 
 1. Download a copy of the `vessel` binary [from the release page](https://github.com/dfinity/vessel/releases) or build one yourself
+   1. For Ubuntu in `$HOME/bin` RUN `wget https://github.com/dfinity/vessel/releases/download/v0.6.1/vessel-linux64` 
```

Comment:
I'd probably tell people to run the `wget`, `chmod +x` and to then `mv` the resulting binary. That way they don't have to be in a specific directory for the instructions to work.

Reply:
yeah that's a good idea, I'll change that :)